src/libs/syscomm/src/lib.rs

```diff
@@ -155,6 +157,71 @@ impl SocketStream {
             },
         }
     }
+
+    /// Read one message from the gateway compising a header with a size
+    /// (u32 LE) and then the body with as many bytes as indicated in the header.
+    pub fn read_message_from_gateway(&mut self) -> Result<Vec<u8>, ::std::io::Error> {
+        let mut buffer = Vec::new();
+        // TODO: define constant
+        let mut tmp = [0u8; 64];
```

Comment:
@ppenna any suggestions on what would be a good constant here?

Reply:
It should be `::config::kernel::IPC_MESSAGE_SIZE`